### PR TITLE
[LAND-525] New Relic keys

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- Clarify that there are two different New Relic integrations (for monitors and APM metrics)
- Clarify that the two integration types use different New Relic API keys
- Add descriptions of health status to "Observing an API" page
- Edit text and update some screenshots on "New Relic" page

### CODE REVIEW

**Review by jkonrath-postman — APPROVED** [review]
LGTM

File: src/pages/docs/designing-and-developing-your-api/observing-an-api.md [hunk 1]
```diff
@@ -70,28 +70,42 @@ New Relic is an application performance management (APM) solution to monitor rea
 
 With this integration, each version of your API in Postman can be linked to multiple services from New Relic. Each service can correspond to a running instance of the API, such as beta, prod1, and prod2. You can also optionally link multiple dashboards from New Relic to your API.
 
+> **You can also send Postman monitor results to New Relic.** Note that sending monitor results to New Relic is a separate integration from viewing APM metrics and uses a different New Relic API key. Learn more about [configuring Postman monitor integration with New Relic](/docs/integrations/available-integrations/new-relic/).
```
Comment (HeatherStenson) [inline src/pages/docs/designing-and-developing-your-api/observing-an-api.md:73]: ❓ 

```suggestion
> **You can also send Postman monitor results to New Relic.** Note that sending monitor results to New Relic is a separate integration from viewing APM metrics and uses a different New Relic API key. Learn more about [configuring a Postman monitor integration with New Relic](/docs/integrations/available-integrations/new-relic/).
```

File: src/pages/docs/designing-and-developing-your-api/observing-an-api.md [hunk 2]
```diff
@@ -70,28 +70,42 @@ New Relic is an application performance management (APM) solution to monitor rea
...
 1. Under **Connect to APM**, select **New Relic**.
 1. On the **Authenticate** tab:
-    1. Enter a nickname.
-    1. Enter a New Relic User API Key. For more information, see the [New Relic documentation](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/).
-    1. Select your New Relic region.
-    1. Select **Authenticate**.
+    * Enter a nickname.
+    * Enter a New Relic User API Key.
+
+        > There are multiple types of API keys in New Relic. Make sure to use a **User** key for connecting an API to New Relic. For more information on API Keys in New Relic, see the [New Relic API keys documentation](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/).
```
Comment (HeatherStenson) [inline src/pages/docs/designing-and-developing-your-api/observing-an-api.md:87]:
```suggestion
        > There are multiple types of API keys in New Relic. Make sure to use a **User** key for connecting an API to New Relic. For more information on API keys in New Relic, see the [New Relic API keys documentation](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/).
```

File: src/pages/docs/integrations/available-integrations/new-relic.md [hunk 3]
```diff
@@ -12,51 +13,56 @@ contextual_links:
...
+If you don't already have a [New Relic account](https://newrelic.com/), you'll need to create one. Then follow the steps below to retrieve your Ingest License key from New Relic:
+
 1. Log in to [New Relic](https://login.newrelic.com).
-1. Select your avatar icon in the upper right of the page.
-1. On the user menu, select **API keys**.
-1. On the **API keys** page, go to the **Not seeing your keys?** heading and click **Insights Insert Keys**.
-  ![New Relic API keys](https://assets.postman.com/postman-docs/new-relic-key-1.jpg)
-1. Select **Show** to the right of an existing key and copy it, or select **+** to create a new key.
-  ![New Relic API keys](https://assets.postman.com/postman-docs/new-relic-key-2.jpg)
+1. Select your user avatar in the upper right corner, and the select **API keys**.
```
Comment (HeatherStenson) [inline src/pages/docs/integrations/available-integrations/new-relic.md:27]:
```suggestion
1. Select your user avatar in the upper right corner, then select **API keys**.
```

**Review by HeatherStenson — APPROVED** [review]
Looks good! 🍊

**Review by shashankawasthi88 — COMMENTED** [review]
Looks good!

**Pull request merged**